etl/steps/data/garden/epa/2024-01-29/ghg_concentration.yml

```diff
@@ -0,0 +1,26 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+      - Climate Change
+    processing_level: major
+
+dataset:
+  update_period_days: 0
+
+tables:
+  ice_sheet_mass_balance:
+    title: Global Atmospheric Concentrations of Greenhouse Gases
+    variables:
+      co2_concentration:
+        title: Global atmospheric concentration of carbon dioxide
```

Comment:
would be useful to have some more context in the metadata here - e.g., `idescription_short: The data come from a variety of historical ice core studies and recent air monitoring sites around the world. `
 
could add other info if available.